src/CodeFormatter/Program.cs

```diff
@@ -46,19 +41,8 @@ private static int Main(string[] args)
         private static int RunExportOptionsCommand(ExportOptions exportOptions)
         {
             int result = FAILED;
-            PropertyBag allOptions = new PropertyBag();
+            PropertyBag allOptions = OptionsHelper.BuildDefaultPropertyBag();
 
-            // The export command could be updated in the future to accept an arbitrary set
-            // of analyzers for which to build an options XML file suitable for configuring them.
-            // Currently, we perform discovery against the built-in CodeFormatter rules
-            // and analyzers only.
-            foreach (IOptionsProvider provider in FormattingEngine.GetOptionsProviders(s_defaultCompositionAssemblies))
-            {
-                foreach (IOption option in provider.GetOptions())
-                {
-                    allOptions.SetProperty(option, option.DefaultValue);
-                }
-            }
             allOptions.SaveTo(exportOptions.OutputPath, id: "codeformatter-options");
```

Comment:
I don't understand -- the export options command now emits an empty property bag into the `codeformatter-options` portion of the specified output file?